70_increase_max_entries.dpatch by Frederic Peters <fpeters@debian.org>

Increase max entries from 1024 to 8192 (closes: #425515, #427476)
--- a/libgphoto2/gphoto2-list.c
+++ b/libgphoto2/gphoto2-list.c
@@ -72,7 +72,7 @@
 
 #ifdef CAMERALIST_STRUCT_COMPATIBILITY
 
-#define MAX_ENTRIES 1024
+#define MAX_ENTRIES 8192
 #define MAX_LIST_STRING_LENGTH 128
 struct _CameraList {
 	int  count;
